lab-server/Dockerfile

```diff
@@ -1,17 +1,24 @@
+# Note: this docker file is meant to be run from the project root as we need to
+# add local modules You can build this image by running `docker build -t
+# lab-server -f ./lab-server/Dockerfile .` from the project root.
+
 FROM node:16
 
 # Create app directory
 WORKDIR /usr/src/app
 
-# Install app dependencies
-COPY package.json ./
-COPY yarn.lock ./
+# Copy top-level package json (needed to register workspaces)
+COPY ./package.json .
+COPY ./yarn.lock .
 
-RUN yarn
+# Copy common module
+COPY ./common ./common
 
-# Bundle app source
-COPY . .
+# Copy lab server
+COPY ./lab-server ./lab-server
```

Comment:
We should probably not copy the entire `lab-server` over rather `package.json`, `yarn.lock` and `requirements.txt`